gnmi_server/server_test.go

```diff
@@ -291,10 +291,16 @@ func runTestSet(t *testing.T, ctx context.Context, gClient pb.GNMIClient, pathTa
 		req = &pb.SetRequest{
 			Prefix: &prefix,
 			Replace: []*pb.Update{&pb.Update{Path: &pbPath, Val: v}},
+			Extension: []*ext_pb.Extension{
+				{Ext: &ext_pb.Extension_MasterArbitration{}},
```

Comment:
If master arbitration is disabled, are these extensions necessary?

Reply:
No. Extension field is optional